### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by satyadvaya**

### CODE REVIEW

File: src/components/Forms/FilterForm.jsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
       <select
         value={selectedType}
         onChange={(event) => filterChange(event.target.value)}
       >
-        <option value='all'>All</option>
-        {types.map(({type}) => (
-          <option value={type}>{type}</option>
+        <option value="all">All</option>
+        {types.map(({ type }) => (
+          <option key={type} value={type}>
```
Comment (katrinkajb) [inline src/components/Forms/FilterForm.jsx:13]: There is one other element on this file that needs a key

File: src/views/Compendium/Compendium.jsx [hunk 2]
```diff
@@ -1,84 +1,94 @@
...
-  //   useEffect(() => {
-  //     const getPokemon = async () => {
-  //       const pokemonList = await fetchPokemon();
-  //        this.setState({pokemons: pokemonList});
-  //       setLoading(false);
-  //     };
-  //     getPokemon();
-  //   }, [pokemons]);
+  useEffect(() => {
+    // const getPokemon = async () => {
```
Comment (katrinkajb) [inline src/views/Compendium/Compendium.jsx:24]: The commented line and this line do the same thing so you could've gone with the original or just gotten rid of the commented out code on line 24

File: src/components/Forms/FilterForm.jsx (hunk 1 above)
Comment (satyadvaya) [reply src/components/Forms/FilterForm.jsx:13]: I've added a key to the option element on line 9 — not sure why React didn't bark at me about that in the console ...

File: src/views/Compendium/Compendium.jsx (hunk 2 above)
Comment (satyadvaya) [reply src/views/Compendium/Compendium.jsx:24]: Thank you for pointing that out!

**Pull request merged**